test/code-fixtures/invalid-lwc/invalidApiDecorator/noLeadingUpperCase.js

```diff
@@ -1,6 +1,8 @@
-import { api } from 'lwc';
+import {api} from "lwc";
```

Comment:
This example code had lots of eslint violations. Updated this one file to be compliant and removed the others. I didn't see the value of such a large test case.